src/hotspot/share/asm/codeBuffer.hpp

```diff
@@ -485,7 +491,7 @@ class CodeBuffer: public StackObj {
     assert(code_start != NULL, "sanity");
     initialize_misc("static buffer");
     initialize(code_start, code_size);
-    verify_section_allocation();
+    debug_only(verify_section_allocation();)
```

Comment:
You also need #ifdef ASSERT around verify_section_allocation() in codeBuffer.cpp.

Reply:
There's still a verification in product code in `CodeBuffer::~CodeBuffer()`. Or do we want to take leap and make this ASSERT-only altogether?

Reply:
But you put debug_only() around declaration here.

Reply:
The declaration is on [line 466](https://github.com/openjdk/jdk/pull/1421/files/7a972eb203ab61054d789d6e8494d90460498b66#diff-deb8ab083311ba60c0016dc34d6518579bbee4683c81e8d348982bac897fe8aeR466). I see that GitHub folds the 2 diffs in this file a bit oddly, so I can see how you mistook this call for the declaration.

Reply:
Shoot! Yes, I mistook it. Changes are good then.